python/lsst/metaserv/schemaToMeta.py

```diff
@@ -260,21 +260,21 @@ def _retrDefaultValue(self, str):
     # "    UNIQUE UQ_AmpMap_ampName(ampName)"
     # "    UNIQUE UQ_x(xx DESC, yy),"
```

Comment:
If you're going to give these as examples, you should include all of them in the tests.  Instead, none of these are included, and the last one still fails.